lib/TransactionRunner.js

```diff
@@ -42,19 +42,43 @@ class TransactionRunner {
   }
 
   run(transactions, callback) {
-    logger.debug('Sorting HTTP transactions');
-    transactions = this.configuration.options.sorted ? sortTransactions(transactions) : transactions;
+    logger.debug('Starting reporters and waiting until all of them are ready');
+    this.emitStart((emitStartErr) => {
```

Comment:
The `emitStart` and `emitEnd` methods were originally part of Dredd.js, but I moved them to the transaction runner. All other events are emitted in the transaction runner, so I thought it's a good idea to streamline the concerns.